docs/architecture/cloud-native/resiliency.md

```diff
@@ -1,39 +1,41 @@
 ---
 title: Cloud-native resiliency
 description: Architecting Cloud Native .NET Apps for Azure | Cloud Native Resiliency
-ms.date: 06/30/2019
+author: robvet
+ms.date: 05/13/2020
 ---
+
 # Cloud-native resiliency
 
-[!INCLUDE [book-preview](../../../includes/book-preview.md)]
+Resiliency is the ability of your system to react to failure and still remain functional. It's not about avoiding failure, but accepting failure as inevitable and crafting  your cloud-native services to respond to it. The end-goal is to return the application to a fully functioning state after a failure.
+
+Cloud-native systems take full advantage of the cloud computing model. Your services must know how to thrive in a dynamic, virtualized cloud environment. They'll make extensive use of Platform as a Service (PaaS) compute infrastructure and managed services. The caveat: Cloud infrastructure and networking are inherently unreliable.
```

Comment:
inherently unreliable is a strong statement and the statement when taken out of context sounds very negative. Can you rephrase to emphasize on transient failures than calling it unreliable? Also, talk about why it is important in a distributed apps to mitigate cascading failures.

Reply:
I felt very disconnected reading the intro.

Reply:
Addressed